Userland/Libraries/LibJS/Runtime/CommonPropertyNames.h

```diff
@@ -82,6 +82,7 @@ namespace JS {
     P(console)                               \
     P(construct)                             \
     P(constructor)                           \
+    P(copyWithin)                            \
```

Comment:
I was wondering if we want to add this since `copyWithin` is a lot more specific than most other strings here?

Reply:
What do you mean by more specific? If it's a property name used by some built-in object, it gets added here.

Reply:
Feels a lot less used then say `constructor` so maybe tiny micro optimization.
But good to know thanks.